### DISCUSSION TITLE
Issue #2124 (BelongsTo sortable() should sort by label) labeled as implemented but not working

### DISCUSSION THREAD

**Discussion by davide-lucchelli** [laravel/nova-issues#5576] [Q&A] [answered]
- Laravel Version: 9.41.0
- Nova Version: 4.19.3
- PHP Version: 8.2.5
- Database Driver & Version: MySql 8.0.33
- Operating System and Version: MacOS 13.2.1 
- Browser type and version: Chrome 113.0.5672.126

### Description:

Trying to sort table by a belongsto column, data is sorted using the related resource id, not the displayed label (behavior normally expected by user).
This behavior was already reported in an issue (https://github.com/laravel/nova-issues/issues/2124) and labeled as "implemented" but in this version is still sorting by id.

**Comment by crynobone** [comment]
https://nova.laravel.com/docs/4.0/resources/relationships.html#disable-ordering-by-title

**Reply by davide-lucchelli** [reply]
This link concerns "select dropdown", I was talking about resource index view sorting functionality

**Reply by crynobone** [reply, marked as answer]
okay, such feature isn't available without introducing performance impact on most app. I removed the implemented tag.

**Reply by davide-lucchelli** [reply]
Ok, I understand that feature could introduce performance impact, but, from the user's point of view, it is a malfunction. It's not the first time I've received complaints from customers for this reason and the only way to solve the situation is to develop a custom sorting using subqueries. I understand the "technological" problem but in fact it is something that does not work as it should.
